taxonomies/food/ingredients.txt

```diff
@@ -39987,27 +39911,19 @@ wikidata:en: Q2461726
 # 132 products in 4 languages @2018-09-28
 
 < en:pear juice
-en: pear juice concentrate
+en: pear juice concentrate, pear juice from concentrate
```

Comment:
not the same thing

Reply:
https://world.openfoodfacts.org/category/pear-juices-from-concentrate
https://world.openfoodfacts.org/category/pear-juice-concentrate

Reply:
This is a stopword problem again.

As there is no pear-juice-concentrate products, I propose to just remove this, and kip pear juice from concentrate.

Reply:
Both are not the same for the calculation of the Nutri-Score right?

`DO CONCENTRATED JUICES COUNT WHEN WORKING OUT “FRUIT, VEGETABLE, PULSES, NUTS, AND
RAPESEED, WALNUT AND OLIVE OILS” CONTENT?
With regards to concentrated fruits and vegetables:
Can be counted: fruit juices made from 100% concentrates
E.g. orange juice made from 100% concentrate that is rehydrated to 100% (or coconut juice when
rehydration results in 100% juice)
Cannot be counted: concentrated fruit juices or fruit puree that have not been rehydrated to 100%
E.g. juice/concentrated lemon syrup used in a sorbet cannot be counted as fruit
`
It's in the NUTRI-SCORE FREQUENTLY ASKED QUESTIONS https://www.royalkoopmans.com/uploads/fckconnector/284da573-50d7-50a8-8eb4-1a51a9a4f84b